polaris.shopify.com/src/components/PatternPage/PatternPage.module.scss

```diff
@@ -127,29 +127,25 @@
 }
 
 .HowItHelps {
-  .ImageWrapper {
+  & > img {
     display: block;
     border: 1px solid var(--p-border-subdued);
-    position: relative;
-    flex: auto;
-    overflow: hidden;
     aspect-ratio: 361.9/180.9;
-    width: 100%;
     border-radius: 0.25rem;
     background: var(--p-surface-depressed);
-    > img {
-      width: 100%;
-      object-fit: fill;
-    }
   }
   ol {
+    --counter-offset: calc(24px + var(--p-space-2));
     list-style: none;
     counter-reset: item;
     li {
       counter-increment: item;
+      padding-left: var(--counter-offset);
       &::before {
         width: 24px;
         height: 24px;
+        position: absolute;
+        margin-left: calc(-1 * var(--counter-offset));
```

Comment:
Simplified ol layout to not use a `<Row>`, so instead we do a negative margin hack.